spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java

```diff
@@ -136,89 +109,6 @@ public JSONObject searchProject(HttpSession session, JSONObject json) {
-    String searchStr = json.getString("str");
-    try {
-      Collection<Pool> pools;
-      if (!isStringEmptyOrNull(searchStr)) {
-        pools = new ArrayList<>(poolService.list(0, 0, false, "id",
-            PaginationFilter.parse(searchStr, SecurityContextHolder.getContext().getAuthentication().getName(), x -> {
-              // Discard errors
-            })));
-      } else {
-        pools = new ArrayList<>(poolService.list(0, 50, false, "id"));
-      }
-
-      StringBuilder b = new StringBuilder();
-      if (pools.size() > 0) {
-        for (Pool p : pools) {
-          generateDashboardCell(b, "pool", p.getId(), p.getName(), p.getAlias());
-        }
-      } else {
-        b.append("No matches");
-      }
-      return JSONUtils.JSONObjectResponse("html", b.toString());
-
-    } catch (IOException e) {
-      log.debug("Failed", e);
-      return JSONUtils.SimpleJSONError("Failed: " + e.getMessage());
-    }
-  }
-
-  public JSONObject searchStudy(HttpSession session, JSONObject json) {
-    String searchStr = json.getString("str");
-    try {
-      List<Study> studies;
-      StringBuilder b = new StringBuilder();
-      if (!isStringEmptyOrNull(searchStr)) {
-        studies = new ArrayList<>(studyService.listBySearch(searchStr));
-      } else {
-        studies = new ArrayList<>(studyService.listWithLimit(50));
-      }
-
-      if (studies.size() > 0) {
-        Collections.sort(studies);
-        Collections.reverse(studies);
-        for (Study s : studies) {
-          generateDashboardCell(b, "study", s.getId(), s.getName(), s.getAlias());
-        }
-      } else {
-        b.append("No matches");
-      }
-      return JSONUtils.JSONObjectResponse("html", b.toString());
-    } catch (IOException e) {
-      log.debug("Failed", e);
-      return JSONUtils.SimpleJSONError("Failed: " + e.getMessage());
-    }
-  }
-
-  public JSONObject searchExperiment(HttpSession session, JSONObject json) {
-    String searchStr = json.getString("str");
-    try {
-      List<Experiment> experiments;
-      StringBuilder b = new StringBuilder();
-      if (!isStringEmptyOrNull(searchStr)) {
-        experiments = new ArrayList<>(experimentService.listAllBySearch(searchStr));
-      } else {
-        experiments = new ArrayList<>(experimentService.listAllWithLimit(50));
-      }
-
-      if (experiments.size() > 0) {
-        Collections.sort(experiments);
-        Collections.reverse(experiments);
-        for (Experiment e : experiments) {
-          generateDashboardCell(b, "experiment", e.getId(), e.getName(), e.getAlias());
-        }
-      } else {
-        b.append("No matches");
-      }
-      return JSONUtils.JSONObjectResponse("html", b.toString());
-    } catch (IOException e) {
-      log.debug("Failed", e);
-      return JSONUtils.SimpleJSONError("Failed: " + e.getMessage());
-    }
-  }
-
   public JSONObject searchRun(HttpSession session, JSONObject json) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "session". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)